package.json

```diff
@@ -1,26 +1,22 @@
 {
   "name": "webrtc-stats-gatherer",
-  "version": "8.0.2",
+  "version": "9.0.0",
   "description": "Gathers stats on interval for webrtc peer connection",
-  "main": "dist/webrtc-stats-gatherer.cjs",
-  "browser": "dist/webrtc-stats-gatherer.min.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
     "src",
     "dist"
   ],
   "scripts": {
-    "lint": "semistandard",
-    "build": "npm run build-web && npm run build-node",
+    "clean": "rimraf dist lib",
+    "build": "npm run clean && tsc",
     "build-web": "webpack && webpack --env.production",
     "build-node": "webpack --env.node && webpack --env.node --env.production",
-    "prepublish": "npm run build && npm run _test",
-    "_test": "_mocha",
-    "test-server": "npm run _test -- -w",
-    "test": "npm run lint && npm run _test",
-    "posttest": "npm run coverage && npm run check:coverage",
-    "coverage": "./scripts/genCoverage.sh",
-    "check:coverage": "./scripts/checkCoverage.sh",
-    "coveralls": "cat ./coverage/lcov.info | coveralls"
+    "_test": "jest",
```

Comment:
NIT: I like `test:unit` over `_test` for running tests without the linting. But that is personal preference.